tests/random_test.py

```diff
@@ -1980,6 +1980,9 @@ def like(keys):
 @skipIf(not config.jax_enable_custom_prng,
         'custom PRNG tests require config.jax_enable_custom_prng')
 class JnpWithKeyArrayTest(jtu.JaxTestCase):
+  def assertKeysEqual(self, key1, key2):
+    self.assertArraysEqual(key1._base_array, key2._base_array)
```

Comment:
```suggestion
    self.assertArraysEqual(key1.unsafe_raw_array(), key2.unsafe_raw_array())
```
We ought to rename this method, but might as well use it consistently until we do.

Reply:
Done, thanks!